README.md

```diff
@@ -67,7 +67,7 @@ Once you have Carthage [installed](#installing-carthage), you can begin adding f
   $(SRCROOT)/Carthage/Build/iOS/ReactiveCocoa.framework
   ```
 
-  This script works around an [App Store submission bug](http://www.openradar.me/radar?id=6409498411401216) triggered by universal binaries.
+  This script works around an [App Store submission bug](http://www.openradar.me/radar?id=6409498411401216) triggered by universal binaries as well as ensures that necessary bitcode-related files are also copied.
```

Comment:
According to the dictionary, "as well as" is defined, when used as a conjunction, as "and in addition", or "and also". Were I to write this as "This script works around an App Store submission bug and also ensures that necessary bitcode-related files are also copied", you'd probably think that looks fine.

In fact, I'll just go ahead and rephrase that. I think it sounds better written that way regardless.